tests/test_geoviewer.py

```diff
@@ -40,9 +41,14 @@ def test_geoviewer_valid_1band(capsys, monkeypatch, filename, option):  # type:
     # To avoid having the plots popping up during execution
     monkeypatch.setattr(plt, "show", lambda: None)
 
+    # The everest example will raise errors
```

Comment:
Why is that? I don't get that warning when calling geoviewer locally on that dataset... Is this a geoutils' warning?

Reply:
Good point! I traced it back, and it's only when the `nodata=99` is used because this calls `set_nodata` in geoviewer, forcing a value that already exists in the raster.
I added a condition to filter the warning only for that specific nodata case in the tests!